ansible_rulebook/util.py

```diff
@@ -380,3 +380,48 @@ def validate_url(url: str, url_type: str) -> bool:
     if url_type == "controller":
         return res.scheme in ["http", "https"] and bool(res.netloc)
     return res.scheme in ["ws", "wss"] and bool(res.netloc)
+
+
+# If the following values exist in a key,
+# replace the value of the key with the MASKED_STRING
+KEYS_TO_FILTER = [
+    "token",
+    "password",
+]
+MASKED_STRING = "******"
+
+
+def mask_sensitive_variable(key: str, val: str):
```

Comment:
@zkayyali812 We need the return type too in the function def.

Reply:
Done